lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -827,13 +829,30 @@ public AuthOptions getAuthOptions() {
      * Renew auth credentials.
      * Will obtain a new token, even if we already have an apparently valid one.
      * Authorization will use the parameters supplied on construction.
+     * @deprecated Use {@link Auth#renewAuth} instead
```

Comment:
I think this entry should be expanded to explain the reasons behind the deprecation, being that this method's behaviour was insufficient for the needs of most developers, given it would return immediately - not providing any mechanism for the caller to be notified when the re-auth had actually completed.

Reply:
89d4acd806e72105801a3f778f5d6e166f782b57